pkgs/applications/misc/electrum/default.nix

```diff
@@ -51,9 +51,14 @@ python3.pkgs.buildPythonApplication {
 
   src = fetchurl {
     url = "https://download.electrum.org/${version}/Electrum-${version}.tar.gz";
-    sha256 = "0yg6ld92a4xgn7y8i51hmr3kmgmrbrjwniikkmyq9q141h2drb80";
+    sha256 = "05m6vbd4sfjk536kwa5wa3kv21jxxqnglx0ddvnmxfhf98371bhk";
   };
 
+  patches = [
+    # Needed to use dnspython 2.1
+    ./relax-dnspython-version.patch
+  ];
+
```

Comment:
You can drop this and the file and replace with this. It seems most of changes like this are done via `substituteInPlace`, not patch.

```
  prePatch = ''
    substituteInPlace contrib/requirements/requirements.txt \
      --replace "dnspython>=2.0,<2.1" "dnspython>=2.0"
  '';
```

Reply:
Done